package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@opencrvs/mosip",
-  "version": "1.8.0-beta.2",
+  "version": "1.8.0-beta.3",
```

Comment:
@tharakadadigama20 you might have to go above the version which is currently released in [npm](https://www.npmjs.com/package/@opencrvs/mosip) or that of [main](https://github.com/opencrvs/mosip/blob/93bb22cd00f17e7ba7a9efc010ffc09ce4ae4f4d/package.json#L3). So you might want to make it  `1.8.0-beta.4` instead?